test/mitmproxy/net/test_tls.py

```diff
@@ -43,7 +43,7 @@ def test_log(self, tmpdir):
 
             tls.log_master_secret.close()
             with open(logfile, "rb") as f:
-                assert f.read().count(b"CLIENT_RANDOM") >= 2
+                assert f.read().count(b"SERVER_HANDSHAKE_TRAFFIC_SECRET") >= 2
```

Comment:
Does this mean the content changed with the new pyopenssl version?
Is Wireshark already compatible with this new format?

Reply:
It definitely changed for TLS 1.3, not sure what we'd get for older versions. It doesn't really matter though, the point of the new API is just that: here are some opaque bytes you should save as-is in your keylog file.

And yes, works fine with WireShark, I used this extensively for sans-io :-)